sbin/build.sh

```diff
@@ -173,8 +173,8 @@ configureVersionStringParameter() {
   local openJdkVersion=$(getOpenJdkVersion)
   echo "OpenJDK repo tag is ${openJdkVersion}"
 
-  # --with-milestone=fcs deprecated at jdk11, removed at jdk12
-  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -lt 12 ]; then
+  # --with-milestone=fcs deprecated at jdk12+ and not used for jdk11- (we use --without-version-pre/opt)
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" == 8 ] && [ "${BUILD_CONFIG[RELEASE]}" == "true" ]; then
     addConfigureArg "--with-milestone=" "fcs"
```

Comment:
This logic will fail for 9 and 10 no?

Reply:
That's the idea as we [shouldn't be doing this on nightly builds for any version](https://github.com/AdoptOpenJDK/openjdk-build/issues/2188#issuecomment-718971081), only releases. If the user wants this to be in the pipeline for whatever reason, they can specify it in via the Jenkins params or `./makejdk-any-platform.sh -C --with-milestone=fcs`